xtrlock.c

```diff
@@ -85,13 +149,10 @@ int main(int argc, char **argv){
   struct timeval tv;
   int tvt, gs;
 
-  if (argc != 1) {
-    fprintf(stderr,"xtrlock: no arguments allowed\n");
-    exit(1);
-  }
-  
   errno=0;  pw= getpwuid(getuid());
```

Comment:
There is no point in this or the following shadow lookup if a custom pw is being used.